### PR TITLE
Various fixes on front end

### PR DESCRIPTION
**Pull request by crdgonzalezca**
# Description

This PR includes several small fixes in front end:
- The UNCode navbar menu option was moved as a hook to be able to translate the different options.
- In my courses while searching a course, allow the user to enter to submit the form. Improves usability.
- Add analytics for contact page.
- Send email in background when several students are registered (`register_students` plugin). Additionally, the list of students that failed to register because the username was either taken or the user was registered with another username.
- When a course is marked as problem bank, but it was eventually deleted, an exception is raised and the available courses to copy tasks from are not shown.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: inginious/frontend/template_helper.py [hunk 1]
```diff
@@ -43,6 +43,8 @@ def __init__(self, plugin_manager, user_manager, default_template_dir, default_l
                                                                                            **kwargs)),
                               "superadmin_options": (
                                   lambda **kwargs: self._generic_hook('superadmin_options', **kwargs)),
+                              "uncode_navbar_options": (
```
Comment (Heldeg) [inline inginious/frontend/template_helper.py:46]: Maybe is a good idea to create a page with a simple explanation about the new hooks

**Review by Heldeg — APPROVED** [review]
LGTM

File: inginious/frontend/plugins/contact_page/pages/constants.py [hunk 2]
```diff
@@ -46,3 +46,8 @@ def get_subject_new_course_id():
 def get_use_minify():
     """ return a boolean to define if use minified files """
     return _use_minified
+
+
+def contact_us_option_hook():
+    return "<li><a href='https://juezun.github.io/' class='navbar-link' target='_blank'>" + \
```
Comment (AndresFelipeCastellanos) [inline inginious/frontend/plugins/contact_page/pages/constants.py]: This item of the UNCode nav element has the same reference as the "About us" page. Not sure if this element needs to redirect to the contact page.

Comment (crdgonzalezca) [reply inginious/frontend/plugins/contact_page/pages/constants.py]: Good catch, thanks!

**Review by AndresFelipeCastellanos — APPROVED** [review]
LGTM

**Pull request merged**